ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1072,7 +1176,7 @@ on('remove:repeating_skill',
     updateTotalCP('skill');
   });
 
-function createBaseSkillsAttributes() {
+function createBaseSkillsAttributes(skip_personal = false) {
```

Comment:
JavaScript syntax should be camelCase.

Reply:
Also this is a double negative scenario.
Instead, `includePersonal = true`, and reverse the if block logic.

Reply:
Changed. And I got rid of the default value.